src/plugins/telemetry/include/plugins/telemetry/telemetry.h

```diff
@@ -120,6 +120,39 @@ class Telemetry : public PluginBase {
         float velocity_down_m_s; /**< @brief Velocity in Down direction in metres/second. */
     };
 
+    /**
+     * @brief Acceleration type in local coordinates.
+     *
+     * The acceleration is represented in the NED (North East Down) frame and in metres/second^2.
+     */
+    struct AccelerationNED {
+        float north_m_ss; /**< @brief Acceleration in North direction in metres/second^2 */
```

Comment:
I think I would call it `north_m_s2` but that's probably just a matter of taste.